core/src/main/scala/com/rackspace/com/papi/components/checker/util/XPathExpressionPool.scala

```diff
@@ -38,8 +40,16 @@ object XPathExpressionPool extends Instrumented {
     val pool = new SoftReferenceObjectPool[XPathExpression](version match { case 1 => new XPathExpressionFactory(expression, nc)
                                                                             case 2 => new XPath2ExpressionFactory(expression, nc)
                                                                          })
-    activeGauges :+ metrics.gauge("Active", expression+" ("+version+")")(pool.getNumActive)
-    idleGauges :+ metrics.gauge("Idle", expression+" ("+version+")")(pool.getNumIdle)
+    Try {
+      activeGauges :+ metrics.gauge("Active", expression+" ("+version+")")(pool.getNumActive)
+    } recover {
+      case e: RuntimeException => logger.info("Problem adding new Active gauge metric.", e)
+    }
+    Try {
+      idleGauges :+ metrics.gauge("Idle", expression+" ("+version+")")(pool.getNumIdle)
+    } recover {
+      case e: RuntimeException => logger.info("Problem adding new Idle gauge metric.", e)
+    }
```

Comment:
Oh, macros! I still haven't used them, so they didn't come to mind, but that seems like a great idea!